Plugins/Flow.Launcher.Plugin.Explorer/Search/WindowsIndex/IndexSearch.cs

```diff
@@ -53,11 +53,11 @@ internal async static Task<List<Result>> ExecuteWindowsIndexSearchAsync(string i
                                                                     dataReaderResults.GetString(0),
                                                                     path,
                                                                     path,
-                                                                    query, true, true));
+                                                                    query, 0, true, true));
                             }
                             else
                             {
-                                fileResults.Add(ResultManager.CreateFileResult(path, query, true, true));
+                                fileResults.Add(ResultManager.CreateFileResult(path, query, 0, true, true));
```

Comment:
Why are we setting the score to 0 in several of the changes, have they always been 0?

Reply:
Yeah, we don't set the score before, which is default 0.